### ISSUE TITLE
archive.extract fails on FreeBSD

### ISSUE THREAD

**Issue by m87carlson** [saltstack/salt#20868]
FreeBSD's default tar, bsdtar, requires -x:

``` 
[INFO    ] Executing state archive.extracted for /opt/
[INFO    ] Executing command "tar z -f '/var/cache/salt/minion/files/base/_opt_splunkforwarder.tar'" in directory '/opt/'
[ERROR   ] Command "tar z -f '/var/cache/salt/minion/files/base/_opt_splunkforwarder.tar'" failed with return code: 1
[ERROR   ] stderr: tar: Must specify one of -c, -r, -t, -u, -x
[ERROR   ] retcode: 1
[ERROR   ] {'pid': 22721, 'retcode': 1, 'stderr': 'tar: Must specify one of -c, -r, -t, -u, -x', 'stdout': ''}

             ----------
              pid:
                  22721
              retcode:
                  1
              stderr:
                  tar: Must specify one of -c, -r, -t, -u, -x
              stdout:
```

**Comment by rallytime** [comment]
@m87carlson Which version of salt are you using? We had a regression revolving around supplying/not-supplying an `x` in the command arguments.

**Comment by m87carlson** [comment]
```
pkg info py27-salt
py27-salt-2014.7.0_1
Name           : py27-salt
Version        : 2014.7.0_1
Installed on   : Thu Feb  5 15:44:53 PST 2015
Origin         : sysutils/py-salt
Architecture   : freebsd:10:x86:64
Prefix         : /usr/local
Categories     : sysutils python
Licenses       : APACHE20
Maintainer     : christer.edwards@gmail.com
WWW            : https://pypi.python.org/pypi/salt
Comment        : Distributed remote execution and configuration management system
```

I apologize, I had not made the connection that 2014.7.2 was release. I'll verify it works there :)

**Comment by rallytime** [comment]
@m87carlson It's all good! Yes - we had a regression in 2014.7.0 regarding this very problem. I think in both 2014.7.1 and 2014.7.2 this has been fixed. Want to give one of those a try and let us know if this is still an issue or not?

**Comment by m87carlson** [comment]
Just verified it is fixed in 2014.7.2 

Sorry for the noise, and thanks again!

**Comment by rallytime** [comment]
No problem! Thanks for using salt! :)
